src/components/Common/StatusBadgeMobile/index.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+  BellIcon,
+  CheckIcon,
+  ClockIcon,
+  MinusSmIcon,
+} from '@heroicons/react/solid';
+import { MediaStatus } from '@server/constants/media';
+
+interface StatusBadgeMobileProps {
+  status?: MediaStatus;
+  is4k?: boolean;
+  indicatorIcon?: React.ReactNode;
+}
+
+const StatusBadgeMobile = ({
+  status,
+  is4k = false,
+  indicatorIcon,
+}: StatusBadgeMobileProps) => {
+  const badgeStyle = ['w-5 rounded-full p-0.5 text-white ring-1'];
+
+  switch (status) {
+    case MediaStatus.PROCESSING:
+      badgeStyle.push('bg-indigo-500 ring-indigo-400');
+      indicatorIcon = <ClockIcon />;
+      break;
+    case MediaStatus.AVAILABLE:
+      badgeStyle.push('bg-green-500 ring-green-400');
+      indicatorIcon = <CheckIcon />;
+      break;
+    case MediaStatus.PENDING:
+      badgeStyle.push('bg-yellow-500 ring-yellow-400');
+      indicatorIcon = <BellIcon />;
+      break;
+    case MediaStatus.PARTIALLY_AVAILABLE:
+      badgeStyle.push('bg-green-500 ring-green-400');
+      indicatorIcon = <MinusSmIcon />;
+      break;
+    default:
+      badgeStyle.push('bg-indigo-500 ring-indigo-400');
+      indicatorIcon = <ClockIcon />;
+  }
+
+  return (
+    <div className="inline-flex whitespace-nowrap rounded-full text-xs font-semibold leading-5 ring-1 ring-gray-700 md:hidden">
```

Comment:
We shouldn't handle hiding this on desktop here. If we want to use these in the TitleCards in the future, this would prevent it. Instead handle whether or not the component is displayed from the place that's using it.

We should probably also rename the component for clarity now that I think about it. Maybe `StatusBadgeMini` is a better name,

Reply:
I think `StatusBadge` is fine... we could add a parameter later for different sizes? 🤔

Reply:
> Maybe we could add a parameter later for different sizes and handle this all within the `StatusBadge` component? 🤔

Not sure if it's necessary since every place we would use this wouldn't need the extra functionality the `StatusBadge` offers, such as linking to the media/etc.

Reply:
That's true :)